lib/rubocop/cop/style/option_hash.rb

```diff
@@ -50,6 +53,11 @@ def on_args(node)
         def supports_keyword_arguments?
           RUBY_VERSION >= '2.0.0'
         end
+
+        def name_in_suspicious_param_names?(arg_name)
+          cop_config.key?('SuspiciousParamNames') &&
+            cop_config['SuspiciousParamNames'].include?(arg_name.to_s)
```

Comment:
I'm operating under the assumption that when the user provides configuration, it takes the user's configuration. Maybe it merges the two, in which case I apologize for the alarm :smile: